povm_toolbox/quantum_info/base_dual.py

```diff
@@ -41,22 +42,15 @@ def get_omegas(
     def is_dual_to(self, frame: BaseFrame) -> bool:
         """Check if `self` is a dual to another frame."""
 
-    @abstractmethod
-    def optimize(self, frame: BaseFrame, **options) -> None:
-        """Optimize the dual to `frame` inplace.
-
-        Args:
-            frame: The primal frame to which ``self`` is a dual.
-            options: keyword arguments specifying how to optimize ``self``.
-        """
-
     @classmethod
     @abstractmethod
-    def build_dual_from_frame(cls, frame: BaseFrame) -> BaseDUAL:
+    def build_dual_from_frame(cls, frame: BaseFrame, alphas: tuple[Any] | None = None) -> BaseDUAL:
         """Construct a dual frame to another frame.
 
         Args:
             frame: The primal frame from which we will build the dual frame.
+            alphas: parameters of the local frame super-operators used to build
+                the local dual frames which form together the product dual frame.
```

Comment:
Please explain why this can be `None` and, if it is, what that implies (e.g. what default will be used).